### PR TITLE
Gcnv qc

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Comment by hanars** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- seqr/migrations/0013_individual_sv_flags.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5829651&bid=19086575)

File: seqr/views/apis/staff_api_2_3_tests.py [hunk 1]
```diff
@@ -788,6 +803,33 @@ def test_upload_qc_pipeline_output(self, mock_file_iter):
...
+            'Found and updated matching seqr individuals for 4 samples'
+        ])
+        self.assertListEqual(response_json['warnings'], ['The following 2 samples were skipped: MANZ_1169_DNA, NA'])
+
+        self.assertIsNone(Individual.objects.get(individual_id='NA19675_1').sv_flags)
+        self.assertListEqual(Individual.objects.get(individual_id='NA19678').sv_flags, ['high_QS_rare_calls:_>10'])
+        self.assertListEqual(Individual.objects.get(individual_id='HG00732').sv_flags, ['raw_calls:_>100'])
+        self.assertListEqual(
+            Individual.objects.get(individual_id='HG00733').sv_flags,
+            ['raw_calls:_>100', 'high_QS_rare_calls:_>10'])
```
Comment (ShifaSZ) [inline seqr/views/apis/staff_api_2_3_tests.py:831]: Is it better to use a `set` comparison?

Comment (hanars) [reply seqr/views/apis/staff_api_2_3_tests.py:831]: no, they are put it in as an ordered list and so there should be no way for the order to ever change

**Comment by hanars** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- seqr/migrations/0013_individual_sv_flags.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5829651&bid=19086575)

**Pull request merged**